pkg/irreparable/irreparable.go

```diff
@@ -0,0 +1,174 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package irreparable
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+
+	"storj.io/storj/internal/migrate"
+	"storj.io/storj/pkg/auth"
+	dbx "storj.io/storj/pkg/irreparable/dbx"
+	pdbauth "storj.io/storj/pkg/pointerdb/auth"
+)
+
+var (
+	mon = monkit.Package()
+)
+
+// Server implements the irreparable RPC service
+type Server struct {
+	DB     *dbx.DB
+	logger *zap.Logger
+}
+
+// RemoteSegmentInfo is info about a single entry stored in the irreparable db
+type RemoteSegmentInfo struct {
+	Key                []byte `protobuf:"bytes,1,opt,name=key,proto3" json:"key,omitempty"`
```

Comment:
What is this Key? Use proper name.

Reply:
key is actually the segment's path info . made the renaming